assets/js/xrp-ledger-toml-checker.js

```diff
@@ -244,16 +255,161 @@ async function validate_address_domain_on_net(address, domain, net) {
   }
 }
 
-function handle_submit(event) {
+function handleSubmitDomain(event) {
   event.preventDefault();
 
   $('.result-title').show()
   $('#result').show()
   $('#log').empty()
 
-  fetch_file()
+  fetchFile()
+}
+
+// ------------------------------------------ DOMAIN VERIFICATION VIA ACCOUNT BELOW ------------------------------------------
+let wallet;
+let socket;
+
+function makeLogEntryWallet(text, raw) {
+  let log
+  if (raw) {
+    log = $('<li></li>').appendTo('#verify-domain-log').append(text)
+  } else {
+    log = $('<li></li>').text(text+" ").appendTo('#verify-domain-log')
+  }
+  log.resolve = function(text) {
+    return $('<span></span>').html(text).appendTo(log)
+  }
+  return log
+}
+
+function fetchWallet() {
+  wallet = $('#verify-domain').val()
+  const checkingLog = makeLogEntryWallet('Checking domain of wallet')
```

Comment:
As mentioned in the [terminology guidelines](https://github.com/XRPLF/xrpl-dev-portal/blob/master/CONTRIBUTING.md#terminology), the term "wallet" has multiple meanings so it's best to use a more precise term. I suggest "Account" for this context.

```suggestion
  const checkingLog = makeLogEntryWallet('Checking domain of account')
```